topics/assembly/tutorials/vgp_workflow_training/tutorial.md

```diff
@@ -207,9 +207,6 @@ After genome profiling, the next step is to run the **VGP HiFi phased assembly w
 >   - {% icon param-file %} "*HiC forward reads*": `3: Hi-C_dataset_F`
 >   - {% icon param-file %} "*HiC reverse reads*": `2: Hi-C_dataset_R`
 >   - {% icon param-file %} "*Genomescope summary dataset*": `19: Genomescope on data 13 Summary`
->   - "*K-mer length*": `31`
->   - "*Ploidy*": `2`
->   - "*Is genome large (>100Mb)?*": `No`
```

Comment:
We need to use this parameter for the yeast, so probably is not a good idea to remove it. Perhaps to include some information about why it is necessary in that case?

Reply:
Hmm, how about I put those parameters mentioned in a sentence that they are set under the hood? I just tried to remove it because someone going through the tutorial was confused when the workflow input page did not have those options available anymore

Reply:
You are right, this is the workflow version; so I'll include this change, but for now let it open until the workflow is updated. Otherwise, the results will diverge.

Reply:
OK! 👍🏼